.github/workflows/ci.yml

```diff
@@ -149,13 +174,11 @@ jobs:
         run: |
           New-Item -Path '${{ matrix.unity-config-path }}' -ItemType Directory
           Set-Content -Path '${{ matrix.unity-config-path }}services-config.json' -Value '${{ secrets.UNITY_LICENSE_SERVER_CONFIG }}'
-
       # Need to write to $GITHUB_PATH to make the environment variable available to other steps.
       - name: Add Unity on PATH
         shell: pwsh
         run: |
           Write-Output '${{ matrix.unity-root }}${{ matrix.unity-version }}/${{ matrix.unity-path }}' | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
-
```

Comment:
the removal of lines 49, 63, 82, 152 and 158 were intentional?

Reply:
not really, probably got auto-formatted when restoring back the functionality. originally this PR was supposed to remove these steps altogether

Reply:
brought the empty lines back